src/components/Dropdown/DropdownButton.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './DropdownButton.css';
+
+class DropdownButton extends Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    if (this.props.disabled) {
+      return;
+    }
+    this.props.onClick();
+    this.props.afterClick();
+  }
+
+  render() {
+    const wrapperDisableStatusClassName = this.props.disabled ? 'g3-dropdown-button__wrapper--disabled' : '';
+
+    const buttonTypeClassName = `g3-dropdown-button__button--${this.props.buttonType}`;
+    const buttonIsSplitClassName = this.props.split ? 'g3-dropdown-button__button--with-split-trigger' : 'g3-dropdown-button__button--without-split-trigger';
+
+    const menuTriggerButtonTypeClassName = `g3-dropdown-button__menu-trigger--${this.props.buttonType}`;
+
+    return (
+      <div
+        ref={this.props.menuTriggerElementRef}
+        className={`g3-dropdown-button__wrapper ${wrapperDisableStatusClassName} ${this.props.className || ''}`}
+      >
+
+        {/* Render dropdown button  */}
+        <button
+          type='button'
+          className={`g3-dropdown-button__button ${buttonIsSplitClassName} ${buttonTypeClassName}`}
+          onClick={this.props.split ? this.handleClick : this.props.handleTriggerMenu}
```

Comment:
just wondering, do we need two separate props for "onClick" and "handleTriggerMenu"? if its a dropdown button shouldn't it open a menu either way, regardless of if its split or not?